### PR TITLE
fix: #31 Fix the defs bug

### PR DESCRIPTION
**Pull request by pastjean**
README.md : update doc to say display none doesnt work in some cases
gulpfile.js : remove display:none injector
inline-svg.html : wrap injected file into hidden div
inline-svg.html : add gradient circle to visually test
index.js : added a global defs , remove inner symbol def

No tests :( sorry

### CODE REVIEW

**Comment by w0rm** [comment]
@pastjean thanks, we definitely have to solve this issue, I can't merge this straight away because we have to add tests and also fix my concern regarding the same ids coming from the same files.

As a disclaimer, I didn't consider this to be used for complex svgs, but once we're getting into nested references, we have to make ids of svg contents unique.

**Comment by pastjean** [comment]
I'll start fixing the simple things right away then the multiple ids shouldnt be to much either!

**Comment by pastjean** [comment]
@w0rm Everything is fixed! Even added a test for `<defs/>`tag on parent.

let's open a new issue for the multiple IDs thing.

File: test.js [hunk 1]
```diff
@@ -119,6 +119,7 @@ describe('gulp-svgstore unit test', function () {
       var result = file.contents.toString()
       var target =
       '<svg xmlns="http://www.w3.org/2000/svg">' +
+      '<defs/>' +
```
Comment (w0rm) [inline test.js]: I wonder if we should not output empty <defs/> in this case.

Comment (pastjean) [inline test.js]: probably , well the grunt one doesnt do , i guessed doing the same was good enough !

**Pull request merged**